### PR TITLE
Update interferon section

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I did some looking to see what else has come out about this. There is probably more we could add if we want to expand on it, but I tried to give a general overview.

### Suggested reviewers (optional)

@aadattoli @cgreene @ajlee21 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
@agitter @cgreene is it possible #842 is causing the build to fail? I was able to build #842 after merging in the changes to rootstock (#840)!

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
...
+Additionally, the methods of administration differed between the two trials, with the SOLIDARITY trial administering IFN-𝛽1a subcutaneously [@doi:10/ftmj].
+The differences in findings between the studies suggests that the method of administration might be relevant to outcomes, with nebulized IFN-𝛽1a more directly targeting receptors in the lungs.
+A trial that analyzed the effect of subcutaneously administered IFN-β-1a on patients with ARDS between 2015 and 2017 had also reported no effect on 28-day mortality [@10.1001/jama.2019.22525], while a smaller study analyzing the effect of subcutaneous IFN administration did find a significant improvement in 28-day mortality for COVID-19 [@doi:10.1128/AAC.01061-20].
+At present, several ongoing clinical trials are investigating the potential effects of IFN-𝛽1a, including in combination with therapeutics such as remdesivir [@clinicaltrials:NCT04492475] and administered via inhalation [@clinicaltrials:NCT04385095].
+Thus, as additional information becomes available, a more detailed understanding of whether and under which circumstances IFN-𝛽1a is beneficial to COVID-19 patients should become more clear.
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:737]:
```suggestion
Thus, as additional information becomes available, a more detailed understanding of whether and under which circumstances IFN-𝛽-1a is beneficial to COVID-19 patients should develop.
```

**Review by cbrueffer — COMMENTED** [review]
Reads well, I just suggested a few minor improvements.

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
...
-They also reported that patients in the treatment group showed greater recovery and lower breathlessness.
-However, given that this information was released in a press release rather than in a manuscript and thus cannot be thoroughly reviewed, these findings should be considered preliminary.
-Additionally, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+These results were subsequently published in November 2020 [@doi:10/ghjzm4].
+The study reports that the participants were assigned at a ratio of 1:1 to receive either SNG001 or a placebo that lacked the active compound, by inhalation for up to 14 days.
+The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at day 15/16.
```
Comment (ajlee21) [inline content/20.pharmaceuticals.md:722]: Just to clarify, does `15/16` mean that the patients were assessed on day 15 or 16 (so 1 or 2 days after treatment ended)? Or is this saying day 15 of the 16-day trial?

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
...
+SNG001 was associated with an odds ratio of 2.32 (95% CI 1.07 – 5.04, _p_ = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, _p_ = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
```
Comment (ajlee21) [inline content/20.pharmaceuticals.md:723]:
```suggestion
SNG001 was associated with an odds ratio (OR) of 2.32 (95% CI 1.07 – 5.04, _p_ = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, _p_ = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
...
+Some of the secondary endpoints analyzed also showed differences: at day 28, the OR for clinical improvement on the OSCI was 3.15 (95% CI 1.39 – 7.14, _p_ = 0.006), and the odds of recovery at day 15/16 and at day 28 were also significant between the two groups.
+Thus, this study suggested that IFN-𝛽1 administered via SNG001 may improve clinical outcomes.
+
+In contrast, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+An obvious difference between these two studies is their sample size, with the WHO Solidarity trial administering IFN-𝛽-1a to 2050 patients and comparing their response to 2050 controls.
+Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; _p_ = 0.11).
```
Comment (ajlee21) [inline content/20.pharmaceuticals.md:729]:
```suggestion
Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; _p_ = 0.11) administering IFN-𝛽-1a to 2050 patients and comparing their response to 2050 controls..
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
...
+The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at day 15/16.
+SNG001 was associated with an odds ratio of 2.32 (95% CI 1.07 – 5.04, _p_ = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, _p_ = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
+Some of the secondary endpoints analyzed also showed differences: at day 28, the OR for clinical improvement on the OSCI was 3.15 (95% CI 1.39 – 7.14, _p_ = 0.006), and the odds of recovery at day 15/16 and at day 28 were also significant between the two groups.
+Thus, this study suggested that IFN-𝛽1 administered via SNG001 may improve clinical outcomes.
+
+In contrast, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+An obvious difference between these two studies is their sample size, with the WHO Solidarity trial administering IFN-𝛽-1a to 2050 patients and comparing their response to 2050 controls.
```
Comment (ajlee21) [inline content/20.pharmaceuticals.md:728]:
```suggestion
```

File: content/20.pharmaceuticals.md (hunk 4 above)
Comment (ajlee21) [reply content/20.pharmaceuticals.md:729]: This is purely a stylistic change and up to you if you want to keep it. It felt a little confusing to me to talk about differences in setup between the two studies that don't necessarily contribute to the differences in the outcome. But then talk about the differences in setup that could lead to differences in outcome later.

**Review by ajlee21 — APPROVED** [review]
Really interesting finding. Just a few minor comments

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:722]:
```suggestion
The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at trial day 15 or 16.
```
It was day 15 or 16, aka a day or two after treatment ended -- so I think the answer to your question is both? haha

**Review by rando2 — COMMENTED** [review]
Thank you @ajlee21!

**Pull request merged**